### PR TITLE
ft(S3C-3000): Add Warp 10 client

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: tests/functional/v2/warp10/testClient.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+const warp10 = new Warp10Client();
+
+const startTime = new Date().getTime();
+const endTime = startTime + 30000; // Add 30 seconds;
+const testValues = generateFakeEvents(startTime, endTime, 1);
+
+const testWarpscript = "CLEAR [ 'OK' ]";
+
+describe.only('Test Warp Client', () => {
```
Comment (rahulreddy) [inline tests/functional/v2/warp10/testClient.js]: Looks like a rouge `.only` made it here.

**Review by miniscruff — CHANGES_REQUESTED** [review]
Just Rahul's comment

**Review by miniscruff — APPROVED** [review]

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**